pkg/controllers/sentinel/kubernetes.go

```diff
@@ -31,6 +31,8 @@ func (h *KVRocksSentinelHandler) ensureKubernetes() error {
 	if err != nil {
 		return err
 	}
+
+	h.pods = []string{}
```

Comment:
Are you intending to explicitly set h.pods to an empty slice, instead of nil? After h.pods has been default initialized, it doesn't seem to have been used. I have tried debugging and have not found the existence of adding duplicate pods. How can I reproduce it?" @rfyiamcool